### PR TITLE
add try catch

### PR DESCRIPTION
**Pull request by vaaaaanquish**
@Hi-king @hirosassa 
fix : #37 
```
HttpError: <HttpError 404 when requesting https://storage.googleapis.com/storage/v1/b/foobar.pkl?alt=json returned "No such object: foobar.pkl". Details: "No such object: foobar.pkl">
```

### CODE REVIEW

**Comment by hirosassa** [comment]
LGTM (need to fixed CI (or Travis itself))

File: thunderbolt/client/local_directory_client.py [hunk 1]
```diff
@@ -17,23 +18,35 @@ def get_tasks(self) -> List[Dict[str, Any]]:
...
+
             tasks_list.append({
                 'task_name': '_'.join(n[:-1]),
                 'task_params': task_params,
                 'task_log': task_log,
                 'last_modified': modified,
                 'task_hash': n[-1].split('.')[0],
             })
+
+        if not_found_log_file_num:
```
Comment (Hi-king) [inline thunderbolt/client/local_directory_client.py:47]: [nits] Or you can also use `len(files) - len(tasks_list)` as not_found_log_file_num

**Comment by Hi-king** [comment]
LGTM

**Review by Hi-king — APPROVED** [review]

File: thunderbolt/client/local_directory_client.py (hunk 1 above)
Comment (vaaaaanquish) [reply thunderbolt/client/local_directory_client.py:47]: Sounds good :)

**Review by hirosassa — APPROVED** [review]

**Comment by vaaaaanquish** [comment]
I will force merge and delete the existing Travis CI. So, enable Github Actions!

**Pull request merged**